### ISSUE TITLE
Feature Request: Able to subscribe to a Pipeline

### ISSUE THREAD

**Issue by sickboy** [ZenHubIO/support#827]
When subscribed to a Pipeline, when an item moves into the Pipeline, I want to receive a notification,
because it's time for me to act, or because I want to be aware of what comes into the pipeline for overview purposes.

**Comment by azenMatt** [comment]
HI @sickboy - do you have a preference of how this notification should come - email, in app - or something else?

**Comment by CRCChelsea** [comment]
@kendramoroz Was that the right issue number? This issue was for subscribing to a pipeline, not filtering a pipeline based on subscription.

**Comment by kendramoroz** [comment]
@CRCChelsea good catch! I crossed wires here with the two issues. Re-opening this one to reflect the difference.

**Comment by mattmcknight** [comment]
Is there still no way to be notified when someone changes the pipeline of an issue that is assigned to you? It's so annoying to not have this. I get notified when anyone does anything else to my issue...
